testsuite/MDAnalysisTests/rng.py

```diff
@@ -0,0 +1,39 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+import numpy as np
+from numpy.testing import TestCase
+import random
+
+seed = 3856349111 # from: np.uint32(hash("MDAnalysis"))
+
+class RandomState(TestCase):
```

Comment:
> Or, in slightly hackish way, MDAnalysisTests.rng could store the np.random.RandomState objects in a module-level dict hashed by the test's class instance. This would emulate class-based caching, without the need for explicitly inheriting anything.

This sounds way to complicated for what we want to do. Why would we need caching anyway?